### ISSUE TITLE
How to change AccessLevel of existing project member?

### ISSUE THREAD

**Issue by Spenhouet** [gmessner/gitlab4j-api#103]
If I have a existing project member, how can I change the `AccessLevel` (user permissions) of this member without removing him from the project? Is this possible?

**Comment by Spenhouet** [comment]
Sorry, I did miss the `getMember` method of the `ProjectApi`.

I found the solution:
`projectApi.getMember(project.getId(), user.getId()).setAccessLevel(AccessLevel.GUEST);`

**Comment by gmessner** [comment]
@Spenhouet 
Setting the access level on the member does not update the member, there is an edit member endpoint but a method to call it is not implemented.  I can add the following if you would like it added to the ProjectApi, simply re-open this issue:
```java
public Member updateMember(Integer projectId, Integer userId, Integer accessLevel) throws GitLabApiException
```

**Comment by Spenhouet** [comment]
@gmessner Good to know. Yes, it would be good to have the ability to change the access level of a project member.

I guess it would be more intuitive if the method would be named:

`public Member updateMemberAccessLevel(Integer projectId, Integer userId, Integer accessLevel)`

**Comment by gmessner** [comment]
We try to mirror the GitLab API as much as possible, which is why GitLab4J is broken down into the sub-apis (ProjectApi, NotesApi, ...). There is actually another setting (expires_at) that can be updated on the member, see:
https://docs.gitlab.com/ee/api/members.html#edit-a-member-of-a-group-or-project

So I will actually be modifying the addMember to include the expiresAt param and creating updateMember() with this signature:

```java
public Member updateMember(Integer projectId, Integer userId, Integer accessLevel, Date expiresAt) throws GitLabApiException
```

**Comment by gmessner** [comment]
Just released 4.7.4.
updateMember() has been added to both the ProjectApi and GroupApi.  Have overloaded methods with and without expiresAt param.
